README.rst

```diff
@@ -39,20 +39,22 @@ There are four key terms or classes you need to understand in order to implement
 
 **Grouper** - an object used to group all versions of an object together - mandatory. This will have a one-to-many relationship with *Content*.
 
-**Version** - an object with a one-to-one relation to a single content record - mandatory. This tracks additional information about that version such as when it was created. You would create this model and it should inherit from *BaseVersion* model included in Versioning models.py
+**Version** - an object with a generic foreign key relation to a single content record - mandatory. This tracks additional information about that version such as when it was created. This model is provided by Versioning.
 
 **Content** - a single versioned object, must have a FK to a *Grouper* object - mandatory. If you are implementing Versioning on an existing content model then this will be that model. All you need to add is the foreign key to your *Grouper* model.
 
-**ContentExtension** - an object which tracks additional information / relationships associated with the content - optional. This can be any model associated with your *Content* model. It may be a 121 or 12M or M2M relationship. It's your responsibility to implement versioning for these extensions via the Version::copy_content method in your implementation of the *Version* class described above.
+**ContentExtension** - an object which tracks additional information / relationships associated with the content - optional. This can be any model associated with your *Content* model. It may be a 121 or 12M or M2M relationship. It's your responsibility to implement versioning for these extensions by providing copy functions in your VersionableItem definitions.
 
 The key things needed in order to implement DjangoCMS Versioning are:
```

Comment:
I, personally, find this quite difficult to read and I find it a bit repetitive as I think you're basically explaining this more clearly with code examples lower down. For me, personally, code examples are always preferable to long blocks of text. So I think we could remove this section completely. What do you think?